### PR TITLE
Fix #396

### LINKED ISSUE #396: Implement the convertStringToBinary function
There is currently a method called `convertStringToBinary` that lives in `/lib/convertStringToBinary.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test convertStringToBinary` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by erickgirard**
Closes #396

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

Please note that I had to fix the tests, the original test had a few typos that made them fails (hence the change in the test file) =)

Added "takeWhile" helper function

### CODE REVIEW

File: lib/takeWhile.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ * `takeWhile` take a slice of an array, elements will be taken 
+ * from the begining of the array until selector returns false
+ * 
+ * @params {Array} array from which values will be iterated
+ * @params {selector} delegate function to determine if we stop iterating
```
Comment (ktilcu) [inline lib/takeWhile.js]: I usually call these functions predicates. If that adds any clarity you are free to change it. If not don't worry about it.

**Comment by ktilcu** [comment]
I will go ahead and merge this now to get it off my plate. if you want to update the docs feel free to submit a new PR. Thanks so much for the contribution and the work. Happy Hacktobering! 🎃
